plugins/hydra_submitit_launcher/tests/test_submitit_launcher.py

```diff
@@ -55,7 +55,35 @@ class TestSubmititLauncher(LauncherTestSuite):
                 },
             },
             ["-m"],
-        )
+        ),
+        # auto queue
+        (
+            {
+                "defaults": [
+                    {"hydra/launcher": "submitit"},
+                    {"hydra/hydra_logging": "hydra_debug"},
+                    {"hydra/job_logging": "disabled"},
+                ],
+                "hydra": {
+                    "launcher": {
+                        "cls": "hydra_plugins.hydra_submitit_launcher.submitit_launcher.SubmititLauncher",
```

Comment:
cls is probably not needed here (and not in where you copied it from).

Reply:
done